corehq/motech/repeaters/management/commands/update_cancelled_records.py

```diff
@@ -106,3 +134,19 @@ def meets_filter(record):
                     time.sleep(float(sleep_time))
 
         print "Wrote log of changes to {}".format(filename)
+
+    def _succeed_record(self, record, success_message, response_status):
+        success_attempt = RepeatRecordAttempt(
+            cancelled=False,
+            datetime=datetime.datetime.utcnow(),
+            success_response=success_message,
+            succeeded=True,
+        )
+        record.add_attempt(success_attempt)
+
+        mock_response = MagicMock()
+        mock_response.status_code = response_status
+        repeater = record.repeater
+        repeater.generator.handle_success(mock_response, repeater.payload_doc(record), record)
```

Comment:
This could be confusing - `handle_success` expects a real server response.  Custom repeaters could be defining their own `handle_success` methods which are meant to deal with a structured response.  For instance, I just found [this repeater](https://github.com/dimagi/commcare-hq/blob/master/custom/enikshay/integrations/nikshay/repeater_generator.py#L133-L149) which expects `response` to have a JSON description with extra information.

I can see that there may be circumstances when we want it called, but I'm not sure this can adequately mock a request.

Reply:
Yeah, I had thought about this. For the repeaters I wanted to run this on, it only cared about the status code, so I went for a YAGNI approach here. Not sure if you think I should make this clearer in the code somehow that this would need to be extended if it is run against other custom repeaters?

Reply:
That's definitely fair.  I think this is fine as long as it breaks in an obvious way if `handle_success` expects something else.  You could also pass in a `--handle_success` flag or something to call it conditionally.  I'm fine leaving it as-is until we need it another way, though.